update - upgrade permissions - SQL Server

dimepues opened the pull request:
Verified sysadmin permissions requirements when upgrading two staging servers from 1.1.443.0 to 1.1.561.0.

-Upgrade failed when only DBO permissions were granted to user running Installation wizard during upgrade
-Upgrade succeeded when sysadmin permissions were granted to user running Installation wizard during upgrade (DBO permissions were already in place)

Recommend internal testing to repro

azurecla commented:
Hi __@ShawnJackson__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-shawja). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

dimepues committed:
update - upgrade permissions - SQL Server

Verified sysadmin permissions requirements when upgrading two staging servers from 1.1.443.0 to 1.1.561.0.

-Upgrade failed when only DBO permissions were granted to user running Installation wizard during upgrade
-Upgrade succeeded when sysadmin permissions were granted to user running Installation wizard during upgrade (DBO permissions were already in place)

Recommend internal testing to repro
